src/test/java/nextstep/subway/section/SectionAcceptanceTest.java

```diff
@@ -0,0 +1,198 @@
+package nextstep.subway.section;
+
+import static nextstep.subway.line.LineAcceptanceTest.*;
+import static nextstep.subway.line.LineAcceptanceTestFixture.*;
+import static nextstep.subway.section.SectionAcceptanceTestFixture.*;
+import static nextstep.subway.station.StationAcceptanceTest.*;
+import static nextstep.subway.station.StationAcceptanceTestFixture.*;
+import static nextstep.subway.utils.JsonPathUtils.*;
+import static org.assertj.core.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.AcceptanceTest;
+
+@DisplayName("지하철역 구간 관련 기능")
+public class SectionAcceptanceTest extends AcceptanceTest {
+    private Integer WANGSIPLI_ID;
+    private Integer JUKJUN_ID;
+
+    private Integer LINE_BUNDANG_ID;
+
+    @BeforeEach
+    public void setUp() {
+        super.setUp();
+        WANGSIPLI_ID = ID_추출(지하철역_생성(WANGSIPLI));
+        JUKJUN_ID = ID_추출(지하철역_생성(JUKJUN));
+
+        // when
+        LINE_BUNDANG_ID = ID_추출(지하철_노선_생성(LINE_BUNDANG, WANGSIPLI_ID, JUKJUN_ID));
```

Comment:
```suggestion
        LINE_BUNDANG_ID_거리_10 = ID_추출(지하철_노선_생성_거리_10(LINE_BUNDANG, WANGSIPLI_ID, JUKJUN_ID));
```

테스트 코드 보는 관점에서 좀더 명확한것 같아요!

Reply:
넵 메서드명에서 픽스처가 명확히 노출되도록 개선하겠습니다!